test/configs/BasicHandlersConfig.ts

```diff
@@ -0,0 +1,72 @@
+import {
+  AllowEverythingAuthorizer,
+  AuthenticatedLdpHandler,
+  BasicResponseWriter,
+  CompositeAsyncHandler,
+  HttpHandler,
+  MethodPermissionsExtractor,
+  QuadToRdfConverter,
+  RawBodyParser,
+  RdfToQuadConverter,
+  ResourceStore,
+  SparqlUpdateBodyParser,
+  SparqlPatchPermissionsExtractor,
+  UnsecureWebIdExtractor,
+} from '../../index';
```

Comment:
@rubensworks you might know this. Originally this said `from '../../'` which caused compilation errors since apparently that caused the compiler to include `index.d.ts` into the builds. Changing it to this fixes it but I'm wondering if there is a tsconfig settings that would also fix it (besides writing the generated files to a different directory).